views/js/controller/creator/views/property.js

```diff
@@ -139,17 +139,27 @@ define(['jquery', 'uikitLoader', 'core/databinder', 'taoQtiTest/controller/creat
          * @private
          */
         function propValidation() {
-            $view.on('validated.group', function (e, isValid) {
-                let $testSection = $('.tlb-button-on').parents('.section').attr('id');
-                let testSectionId = `#${$testSection}`;
-                let $propsSectionError = $(`#section-props-${$testSection}`).find('span.validate-error');
-                let $propsItemError = $('.itemref-props').find('span.validate-error');
-
-                if (e.namespace === 'group') {
-                    if (isValid && $propsItemError.length === 0 && $propsSectionError.length === 0) {
-                        $(testSectionId).removeClass('section-error');
+            $view.on('validated.group', function(e, isValid){
+                const $warningIcon = $('span.icon-warning');
+                const $test = $('.tlb-button-on').parents('.test-creator-test');
+
+                // finds error current element if any
+                let errors = $(e.currentTarget).find('span.validate-error');
+                let currentTargetId = '#' + $(e.currentTarget).find('span[data-bind="identifier"]').attr('id').slice(6);
```

Comment:
```suggestion
                let currentTargetId = '[id="' + $(e.currentTarget).find('span[data-bind="identifier"]').attr('id').slice(6) + '"]';
```
or better use template strings
and jquery has issue to select id with dot , example '#ab.cd', should be used [id="ab.cd"]
and now in id allowed dots